sklearn/model_selection/_split.py

```diff
@@ -997,6 +997,16 @@ def split(self, X, y=None, groups=None):
             for train_index, test_index in cv.split(X, y, groups):
                 yield train_index, test_index
 
+    def get_n_splits(self):
```

Comment:
@jnothman the calculation for get_n_splits function for RepeatedKFold and RepeatedStratifiedKFold depends only on n_splits and n_repeats and is independent of X or groups or y.  Should I send the arguments -> n_splits and n_repeats in the get_n_splits function